client/extensions/woocommerce/woocommerce-services/state/shipping-label/reducer.js

```diff
@@ -126,14 +126,17 @@ reducers[ WOOCOMMERCE_SERVICES_SHIPPING_LABEL_TOGGLE_STEP ] = ( state, { stepNam
 };
 
 reducers[ WOOCOMMERCE_SERVICES_SHIPPING_LABEL_UPDATE_ADDRESS_VALUE ] = ( state, { group, name, value } ) => {
+	const isAddressField = name !== 'name' && name !== 'company' && name !== 'phone';
 	const newState = { ...state,
 		form: { ...state.form,
 			[ group ]: { ...state.form[ group ],
 				values: { ...state.form[ group ].values,
 					[ name ]: value,
 				},
-				isNormalized: false,
-				normalized: null,
+				...( isAddressField && {
+					isNormalized: false,
+					normalized: null,
+				} ),
```

Comment:
Does this not cause errors?

If `false` ends up in the spread, it should error with `TypeError: false is not iterable`.

Reply:
I believe that's true for arrays / args but for objects it need not be iterable, and passing a primitive type is the same as no spread at all AFAIK — I can only find [official mention](https://github.com/tc39/proposal-object-rest-spread/blob/master/Spread.md) of `null` and `undefined` being ignored though so I'll fix.